_episodes/06-func.md

```diff
@@ -374,19 +374,58 @@ we can break the string across multiple lines:
 def offset_mean(data, target_mean_value):
     '''Return a new array containing the original data
        with its mean offset to match the desired value.
-    Example: offset_mean([1, 2, 3], 0) => [-1, 0, 1]'''
+
+    Parameters
+    ----------
+    data : numpy array, or list
+        The data we want to correct the average
+    target_mean_value: float or int
+        The target mean value we want for our new array
+
+    Returns
+    -------
+    numpy array: 
+        Same dimension as `data`, with mean value `target_mean_value`
+
+    Examples
+    --------
+    >>> offset_mean([1, 2, 3], 0)
+    array([-1.,  0.,  1.])
+    '''
     return (data - numpy.mean(data)) + target_mean_value
 
 help(offset_mean)
 ~~~
 {: .language-python}
 
+We're also the [numpy docstring
+standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard),
+which is common across the scientific Python ecosystem and understood by many
+tools. 
+
 ~~~
-Help on function center in module __main__:
+Help on function offset_mean in module __main__:
```

Comment:
This is a change we should include. Would you be willing to submit this naming correction in a separate PR?